### PR TITLE
Hide elements during non-school days

### PR DESCRIPTION
**Pull request by nickthegroot**

### CODE REVIEW

File: classtime.js [hunk 1]
```diff
@@ -214,17 +325,25 @@ function updateVariables() {
...
     document.getElementById('time').innerHTML = getCurrentTimeString();
     document.getElementById('date').innerHTML = getCurrentDateString();
+    
+    if (currentClassPeriodIndex == -1) {
+        document.getElementById("schedule").innerHTML = "and there's <strong>no class</strong> today!"
+        $('.schoolDays').css('display', 'none')
+        return
+    }
+
+    $('.schoolDays').css('display', 'inline')
```
Comment (MoralCode) [inline classtime.js]: why is jQuery needed for this? iirc you can just use something like this (probably not totally valid but) `document.getGetElementByClassName("schoolDays").style.display = "inline";`

File: classtime.js [hunk 2]
```diff
@@ -1,210 +1,321 @@
...
-//nextClassPeriodIndex
-
-var currentScheduleIndex = -1;
-
-
-var use24HourTime = getLocalStorageBoolean("use24HourTime", false);
-
-var data = {
-    fullName: "",
-    shortName: "",
```
Comment (MoralCode) [inline classtime.js]: why did the fullname and Shortname get removed? these were for storing information about the school so it could be displayed on the schedule page once we moved the data to a database and had multiple schools

File: classtime.js [hunk 3]
```diff
@@ -1,210 +1,321 @@
...
+var currentSeconds = 0;
+var currentDate;
 
+var currentClassPeriodIndex = -1;
+var currentScheduleIndex = -1;
+var use24HourTime = getLocalStorageBoolean("use24HourTime", false);
 
 function update() {
     updateTime();
-    if (typeof data !== 'undefined') {
```
Comment (MoralCode) [inline classtime.js]: why was this removed? it is supposed to check if the data exists (mostly for once it is moved to a database)

File: classtime.js [hunk 4]
```diff
@@ -214,17 +325,25 @@ function updateVariables() {
...
+    }
+
+    $('.schoolDays').css('display', 'inline')
     document.getElementById("schedule").innerHTML = "You are viewing the <strong>" + getCurrentScheduleName() + "</strong> schedule."
+    document.getElementById('timeToEndOfClass').innerHTML =  getTimeToEndOfCurrentClassString()
+    document.getElementById("nextClass").innerHTML = getClassName(currentClassPeriodIndex+1)
+    document.getElementById("currentClass").innerHTML = getClassName(currentClassPeriodIndex)
 }
 
 function getCurrentScheduleName() {
```
Comment (MoralCode) [inline classtime.js]: this method is only ever called if the schedule index is not -1, so adding a `return "no class"` wont do anything

**Comment by MoralCode** [comment]
there seems to be a lot more than just UI changes in this commit

**Review by MoralCode — COMMENTED** [review]
there seems to be a lot more than just UI changes in this commit

File: classtime.js (hunk 1 above)
Comment (MoralCode) [reply classtime.js]: implemented in dev branch

**Comment by MoralCode** [comment]
@nbd9 you okay with me closing this PR since hiding elements during non-school days has already been implemented on the dev branch?

**Comment by nbd9** [comment]
No complaints here. Feel free to remove.
